src/Domain/DomainHelpers.hpp

```diff
@@ -61,6 +61,19 @@ void set_internal_boundaries(
         std::vector<DirectionMap<VolumeDim, BlockNeighbor<VolumeDim>>>*>
         neighbors_of_all_blocks) noexcept;
 
+/// \ingroup ComputationalDomainGroup
+/// Sets up the BlockNeighbors using the corner numbering scheme
+/// implied by the maps provided by the user to deduce the correct
+/// neighbors and orientations. Does not set up periodic boundary
```

Comment:
This is a long "brief" comment. Perhaps split the second sentence off and making it a `\warning` since that's a fairly important thing to be aware of :)

Reply:
Did you want to do this? Don't feel like you have to, just a suggestion :)

Reply:
Sorry, I missed this...